### PR TITLE
Suggestions on unit 1

### PR DESCRIPTION
**Pull request by guiyrt**
Hello 😃 Just a couple suggestions/fixes after finishing Unit 1:

1. [c3a210f] On "The Reinforcement Learning Framework" page, it states that in Super Mario there are 5 possible actions: 4 directions and jumping. Isn't it 4 directions including jumping?

2. [ec89732] I noticed some duplicate paragraphs directly below some figure captions in a few places, is this intended?

Let me know what you think, just trying to help refine the experience for everyone 🤗

### CODE REVIEW

File: units/en/unit1/rl-framework.mdx [hunk 1]
```diff
@@ -61,8 +61,6 @@ In a chess game, we have access to the whole board information, so we receive a
 <figcaption>In Super Mario Bros, we only see the part of the level close to the player, so we receive an observation.</figcaption>
 </figure>
 
-In Super Mario Bros, we only see the part of the level close to the player, so we receive an observation.
```
Comment (simoninithomas) [inline units/en/unit1/rl-framework.mdx:64]: This repetition is intended, one is the text the other the figcaption (and given most people don't read figcaption 😆 )

File: units/en/unit1/rl-framework.mdx [hunk 2]
```diff
@@ -83,12 +81,10 @@ The actions can come from a *discrete* or *continuous space*:
 
 <figure>
 <img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/mario.jpg" alt="Mario">
-<figcaption>Again, in Super Mario Bros, we have only 5 possible actions: 4 directions and jumping</figcaption>
+<figcaption>In Super Mario Bros, we have only 4 possible actions: left, right, jumping and ducking</figcaption>
```
Comment (simoninithomas) [inline units/en/unit1/rl-framework.mdx:84]: Can you change ducking to jumping? It's easier to understand for most readers

**Review by simoninithomas — CHANGES_REQUESTED** [review]
Thanks for your PR 🤗 

I added some comments but I was wondering if you can put again the two repetitions (line 63, 87 and line 85 in two-methods.mdx)

File: units/en/unit1/rl-framework.mdx (hunk 2 above)
Comment (guiyrt) [reply units/en/unit1/rl-framework.mdx:84]: By ducking I meant crounching, opposite direction of jumping. Should I just change it to left, right, up and down? "Up" would be jumping and "down" would be ducking/crouching. I could also place between brackets the action associated with the direction. Being that said, the options are:

**In Super Mario, we only have 4 possible actions: ...**

1. left, right, up and down.
2. left, right, up (jumping) and down (crouching).


If I got something wrong, leave a third suggestion and I'll go with that :)

Comment (simoninithomas) [reply units/en/unit1/rl-framework.mdx:84]: I like this one : left, right, up (jumping) and down (crouching). Thanks 🤗

**Comment by guiyrt** [comment]
Just updated the PR based on your input, anything else let me know 🤗

**Comment by simoninithomas** [comment]
Perfect thanks I merge it 🤗

**Pull request merged**